Resources/public/Admin.js

```diff
@@ -608,6 +608,21 @@ var Admin = {
                 lessLink: '<a href="#">'+jQuery(this).data('readmore-less')+'</a>'
             });
         });
+    },
+    handle_inline_delete_checkboxes: function() {
+        var checkboxes = jQuery('.sonata-admin-type-delete-checkbox');
+
+        if(checkboxes.length) {
+            var eventType = window.SONATA_CONFIG.USE_ICHECK ? 'ifChanged': 'change';
+
+            checkboxes.on(eventType, function() {
```

Comment:
http://stackoverflow.com/questions/12824549/should-all-jquery-events-be-bound-to-document

Still thinking that it is better to bind it to document. Reasons explained on that link. 

WDYT? @sonata-project/contributors